src/tools/kdb/shell.cpp

```diff
@@ -60,6 +62,25 @@ int ShellCommand::execute (Cmdline const &)
 			Key parentKey (parent, KEY_END);
 			cout << "return value: " << kdb.set (current, parentKey) << endl;
 		}
+		else if (command == "kdbMerge")
+		{
+			string theirRootString;
+			is >> theirRootString;
+			Key theirRootKey (theirRootString, KEY_END);
+			KeySet their;
+			kdb.get (their, theirRootKey);
+			ckdb::KeySet * result;
```

Comment:
The result is discarded? What can a user do with the kdbMerge command?

Reply:
The result is appended to the current key set 5 lines later. Users could then use kdbSet. As #3131 didn't specify anything about the shell integration and I have never used `kdb shell` before I just implemented something that looked sort of useful to me.

Reply:
ahh, ok. It is a bit hidden within the cout statement. Can you refactor this a bit. Furthermore the output should be more like "error in merge: ..." and not "return value", which is hard to understand if you do not know the API.